workflow/schemas/resources.schema.yaml

```diff
@@ -20,43 +15,71 @@ properties:
       partition:
         type: string
         description: default partition to use on the cluser for a rule
+      threads:
+        type: integer
+        description: default number of threads that will be used by a rule
+      time:
+        type: string
+        description: default max execution time for a rule
   required:
     - default_resources
 
   gene_fuse:
     type: object
+    description: resource definitions for gene_fuse
     properties:
       threads:
         type: integer
-        description: number of threads that will be used by 'gene_fuse'
+        description: number of threads that will be used by rule
```

Comment:
I think you can skip "for rule" and "by rule". It is somehow implied that it is connected to a specific rule by the hierarchical order.